.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         uses: lukka/run-vcpkg@v11.4
         with:
           vcpkgDirectory: '${{ github.workspace }}/vcpkg'
-          vcpkgGitCommitId: 1be4527b3f30ab4fd01b621a3fc355b49b995ad0 
+          vcpkgGitCommitId: a34c873a9717a888f58dc05268dea15592c2f0ff
```

Comment:
how would I go about verifying this is a correct/good commit hash?

Reply:
It's the hash for the latest tag on the vcpkg repo

Reply:
cool, found it: https://github.com/microsoft/vcpkg/commit/a34c873a9717a888f58dc05268dea15592c2f0ff